package/network/services/hostapd/files/hostapd.sh

```diff
@@ -392,13 +392,15 @@ hostapd_set_bss_options() {
 			set_default reassociation_deadline 1000
 			set_default pmk_r1_push 0
 			set_default ft_psk_generate_local 0
+			set_default ft_over_ds 0
```

Comment:
The hostapd documentation says this is activated by default:
```
# Whether to enable FT-over-DS
# 0 = FT-over-DS disabled
# 1 = FT-over-DS enabled (default)
#ft_over_ds=1
```
Why do you deactivate it by default now?